examples/todo-list-plain-js/README.md

```diff
@@ -0,0 +1,96 @@
+
+
+# Supabase Project Documentation
```

Comment:
For the readme file, it would be great if you could follow the same format as other todo-list example projects as well. 
https://github.com/supabase/supabase/tree/master/examples/todo-list/sveltejs-todo-list

Reply:
And let's add this example app under `examples/todo-list` directory.